app/src/app/components/Map.tsx

```diff
@@ -173,7 +192,16 @@ export const MapComponent: React.FC = () => {
   useEffect(() => {
     if (mapLoaded && map.current && zoneAssignments.size) {
       const assignments = FormatAssignments();
-      patchUpdates.mutate(assignments);
+      if (activeTool === "brush") {
```

Comment:
H: Because we want the backend document to remain consistent with the map shown to the user, I'm hesitant to filter assignments sent to the backend here for the time being. 

This is another good reason to accept null zone assignments and use the same endpoint–`api/update_assignments`–for now if we can.

Reply:
:+1: